src/lib.rs

```diff
@@ -109,6 +111,44 @@ pub fn decode_to_array<const N: usize>(hex: &str) -> Result<[u8; N], DecodeFixed
     }
 }
 
+/// Encodes `data` as hex string using lowercase characters.
+///
+/// Lowercase characters are used (e.g. `f9b4ca`). The resulting string's length is always even,
+/// each byte in `data` is always encoded using two hex digits. Thus, the resulting string contains
+/// exactly twice as many bytes as the input data.
+///
+/// # Example
+///
+/// ```
+/// use hex_conservative as hex;
+/// assert_eq!(hex::encode_lower("Hello world!"), "48656c6c6f20776f726c6421");
+/// assert_eq!(hex::encode_lower(vec![1, 2, 3, 15, 16]), "0102030f10");
+/// ```
```

Comment:
I stole all these docs from the [hex](https://docs.rs/hex/latest/hex/) crate - I believe that is within the legality of the licence since it was once said docs are not copyright.

Reply:
Docs are definitely copyrighted. We cannot pull stuff out of a MIT-licensed crate into a CC0 crate without permission.